xarray/backends/zarr.py

```diff
@@ -1397,30 +1444,18 @@ def _get_open_params(
     if isinstance(store, os.PathLike):
         store = os.fspath(store)
 
-    if zarr_version is None:
-        # default to 2 if store doesn't specify it's version (e.g. a path)
-        zarr_version = getattr(store, "_store_version", 2)
-
     open_kwargs = dict(
         # mode='a-' is a handcrafted xarray specialty
         mode="a" if mode == "a-" else mode,
         synchronizer=synchronizer,
         path=group,
     )
     open_kwargs["storage_options"] = storage_options
-    if zarr_version > 2:
-        open_kwargs["zarr_version"] = zarr_version
-
-        if consolidated or consolidate_on_close:
-            raise ValueError(
-                "consolidated metadata has not been implemented for zarr "
-                f"version {zarr_version} yet. Set consolidated=False for "
-                f"zarr version {zarr_version}. See also "
-                "https://github.com/zarr-developers/zarr-specs/issues/136"
-            )
 
-        if consolidated is None:
-            consolidated = False
+    if _zarr_v3():
```

Comment:
zarr-python 3.x changed this parameter name. xarray should probably go through the same deprecation from zarr_version to zarr_format.

Reply:
It may be worth copying over this function: 

https://github.com/zarr-developers/zarr-python/blob/b1c4e4725a36c61dbc18bd01b664ce1589191b75/src/zarr/api/asynchronous.py#L114-L127